bn-apps/memberships-management/membership-service-contracts-and-states/src/main/kotlin/com/r3/businessnetworks/membership/states/Schemas.kt

```diff
@@ -1,23 +1,25 @@
-package com.r3.businessnetworks.membership.states
-
-import net.corda.core.identity.Party
-import net.corda.core.schemas.MappedSchema
-import net.corda.core.schemas.PersistentState
-import net.corda.core.serialization.CordaSerializable
-import javax.persistence.Column
-import javax.persistence.Entity
-import javax.persistence.Table
-
-@CordaSerializable
-object MembershipStateSchemaV1 : MappedSchema(schemaFamily = MembershipState::class.java, version = 1, mappedTypes = listOf(PersistentMembershipState::class.java)) {
-    @Entity
-    @Table(name = "membership_states")
-    class PersistentMembershipState (
-            @Column(name = "member_name")
-            var member: Party,
-            @Column(name = "bno_name")
-            var bno: Party,
-            @Column(name = "status")
-            var status: MembershipStatus) : PersistentState()
-}
-
+package com.r3.businessnetworks.membership.states
+
+import net.corda.core.identity.Party
+import net.corda.core.schemas.MappedSchema
+import net.corda.core.schemas.PersistentState
+import net.corda.core.serialization.CordaSerializable
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.Table
+
+@CordaSerializable
+object MembershipStateSchemaV1 : MappedSchema(schemaFamily = MembershipState::class.java, version = 1, mappedTypes = listOf(PersistentMembershipState::class.java)) {
+    @Entity
+    @Table(name = "membership_states")
+    class PersistentMembershipState(
+            @Column(name = "member_name")
+            var member: Party,
+            @Column(name = "bno_name")
+            var bno: Party,
+            @Column(name = "networkID")

```

Comment:
I'm unsure what the policy on schema changes is in this project. If people were to upgrade their BNMS CorDapp this would probably break and a migration would be necessary.